include/dlaf/lapack_tile.tpp

```diff
@@ -8,6 +8,24 @@
 // SPDX-License-Identifier: BSD-3-Clause
 //
 
+template <class T, Device device>
+dlaf::BaseType<T> lange(lapack::Norm norm, const Tile<T, device>& a) noexcept {
+  return lapack::lange(norm, a.size().rows(), a.size().cols(), a.ptr(), a.ld());
+}
+
+template <class T, Device device>
+dlaf::BaseType<T> lantr(lapack::Norm norm, blas::Uplo uplo, blas::Diag diag,
+                        const Tile<T, device>& a) noexcept {
+  return lapack::lantr(norm, uplo, diag, a.size().rows(), a.size().cols(), a.ptr(), a.ld());
+}
+
+template <class T, Device device>
+void potrf(blas::Uplo uplo, const Tile<T, device>& a) {
+  auto info = potrfInfo(uplo, a);
+  if (info != 0)
+    throw std::runtime_error("Error: POTRF: A is not positive definite.");
```

Comment:
Are we expecting people to recover from this? If not, we should use an `DLAF_ASSERT*`.

Reply:
As stated in another PR, I'm leaving it for the dedicated PR #178 so it would be easier to merge the work later.

My idea about it is to not "alter" the existing code but at the same time do the right thing when adding something new.

Do you agree?

Reply:
Yeah, certainly!